libs/wire-api/src/Wire/API/Provider/Service.hs

```diff
@@ -205,6 +208,22 @@ data Service = Service
   }
   deriving stock (Eq, Show, Generic)
   deriving (Arbitrary) via (GenericUniform Service)
+  deriving (S.ToSchema) via (Schema Service)
```

Comment:
please also remove the manual `{To,From}JSON` instances, or leave a comment why that's hard.